plugins/jobs/src/js/components/form/reducers/ContainerReducers.ts

```diff
@@ -0,0 +1,148 @@
+import { deepCopy } from "#SRC/js/utils/Util";
+
+import { JobSpec, JobFormActionType } from "../helpers/JobFormData";
+
+export const cmdOnlyReducers = {
+  [JobFormActionType.Set]: (value: "true" | "false", state: JobSpec) => {
+    const stateCopy = deepCopy(state);
+    if (value === "true") {
+      stateCopy.cmdOnly = true;
+    }
+    if (value === "false") {
+      stateCopy.cmdOnly = false;
+    }
```

Comment:
The problem with throwing an error here is that the user experience will crash instead of give them something useful - for that reason I would rather fail silently. But it really should never be anything other than "true" or "false", because if it's coming through this reducer then it has been edited in a form field, which we have control over.

Reply:
Ok I'm convinced :) I've changed this to throw if the value isn't `true` or `false`!